Exports/_export/_modules/custom-type/products.php

```diff
@@ -4,123 +4,125 @@
 
 	OUTPUT PRODUCTS AS CSV
 
-	How to use:
-	Upload this file to your site root, and use this URL to download a CSV:
-	http://www.site.com/export-products.php
-
 	*/
 
 	require('../../_inc/config.php');
 
-	$filename = getSiteId() . '_' . 'products' . 'Export' . date('M') . '_' . date('d') . '_' . date('Y');
-	$howmany = 100; // Set to number of products in the module
+	set_time_limit(0);
 
-	// Header
+    // Header
 	header("Content-type: text/csv");
 	header("Content-Disposition: attachment; filename=" . $filename . ".csv");
 	header("Pragma: no-cache");
 	header("Expires: 0");
-
+	
 	// Functions
-	function processItem($in){
+	function processItem($in)
+	{
 		$out = trim($in);
 		$out = str_replace('"','""',$out);
 		$out = str_replace('&amp;','&',$out);
 		$out = '"' . $out . '"';
+
 		return $out;
 	}
 
+	$filename = getSiteId() . '_' . 'products' . 'Export' . date('M') . '_' . date('d') . '_' . date('Y');
```

Comment:
The filename is only called in the `header` calls which happen before you define filename here.  While the `howmany` line can be moved into the function, `$filename` needs to say above the header calls.

Reply:
Gotcha, fixed in commit 9726422